u3-views/features/videoController.js

```diff
@@ -0,0 +1,20 @@
+const { getVideo,getSuggestions } = require('../_services/fakeapi')
+
+const videoDetails1 = (req, res) => {
+  const videoId = req.params.code
+  const video = getVideo(videoId)
+  const teases = getSuggestions(6)
+res.render('video-player', { video: video, teases: teases })
+}
+
+const videoDetails2 = (req, res) => {
+  const videoId = req.params.code
```

Comment:
สิ่งนี้ต้องตรงกับการประกาศเส้นทางของคุณใน `app.js` หากคุณประกาศเส้นทางเป็น `/videos/:videoId` คุณจะได้รับค่าโดยใช้ `req.params.videoId`

This needs to match your route declaration in `app.js`. If you declare the route as `/videos/:videoId`, then you can get the value using `req.params.videoId`.